WordPress/build.gradle

```diff
@@ -442,7 +442,8 @@ task violationCommentsToGitHub(type: se.bjurr.violations.comments.github.plugin.
 {{violation.message}}
 """
     violations = [
-            ["CHECKSTYLE", ".", ".*/build/.*\\.xml\$", "Checkstyle"]
+            ["CHECKSTYLE", ".", ".*/build/.*\\.xml\$", "Checkstyle"],
```

Comment:
Question (❓): Wouldn't it better if we update the `CheckStyle` configuration to a less generic path, just like we do for `Detekt`?

For example: `["CHECKSTYLE", ".", ".*/build/.*/checkstyle/.*\\.xml\$", "CheckStyle"]`

I am actually not sure how this path is being traversed by the `Violation Comments` plugin, but I worry that this regular expression might resolve into multiple paths. I speculate that the reason it works atm might be that even though this regular expression resolves into multiple paths, the first result is being taken into consideration. And, since the first result ends up being the `checkstyle.xml` file it works. But, this might break in the future. Wdyt?

Reply:
Good point!
I've pushed a commit to update the filter.

Reply:
Great @loremattei , many thanks!

I reviewed this commit and it LGTM, I will now approve and merge this PR.